qa/suites/crimson-rados/rgw/singleton/objectstore

```diff
@@ -0,0 +1 @@
+.qa/objectstore_cephfs
```

Comment:
Why is `objectstore_cephfs` used here? We could probably use qa/objectstore to tests against bluestore and seastore.